cmd/manager.go

```diff
@@ -804,7 +818,7 @@ func startJQ(postCreation []byte) {
 // so that the next time the manager is started it will create a new token.
 // For un-clean exits of the manager, we should keep the token so the manager
 // re-uses it, allowing any runners to reconnect.
-func deleteToken() {
+func deleteToken(ctx context.Context) {
```

Comment:
ctx not used?

Reply:
Yes, all these ctx were mainly to pass to die,info,warn methods of root file and now not needed. 
I have updated all these and also removed other unnecessary ctx.